pkg/cmd/controller/build.go

```diff
@@ -65,11 +83,13 @@ func RunBuildController(ctx *ControllerContext) (bool, error) {
 		KubeClient:                         externalKubeClient,
 		BuildClient:                        buildClient,
 		DockerBuildStrategy: &buildstrategy.DockerBuildStrategy{
-			Image: imageTemplate.ExpandOrDie("docker-builder"),
+			Image:                    imageTemplate.ExpandOrDie("docker-builder"),
+			BuildCSIVolumesAvailable: foundBuildCSIVolumes && foundCSIDriverSharedResource,
 		},
 		SourceBuildStrategy: &buildstrategy.SourceBuildStrategy{
-			Image:          imageTemplate.ExpandOrDie("docker-builder"),
-			SecurityClient: securityClient.SecurityV1(),
+			Image:                    imageTemplate.ExpandOrDie("docker-builder"),
+			SecurityClient:           securityClient.SecurityV1(),
+			BuildCSIVolumesAvailable: foundBuildCSIVolumes && foundCSIDriverSharedResource,
```

Comment:
Only check the build flag:
```suggestion
			BuildCSIVolumesAvailable: foundBuildCSIVolumes,
```

Reply:
updated @adambkaplan PTAL